openshift-hack/cmd/k8s-tests-ext/k8s-tests.go

```diff
@@ -88,7 +89,19 @@ func main() {
 		Qualifiers: []string{withExcludedTestsFilter(`name.contains('[Serial]')`)},
 	})
 
+	// ignoredImagePrefixes we don't want to mirror, i.e. things that are intentionally invalid.
+	ignoredImagePrefixes := []string{
+		"invalid.registry.k8s.io/invalid",
+	}
+
+imageLoop:
 	for k, v := range image.GetOriginalImageConfigs() {
+		for _, i := range ignoredImagePrefixes {
+			if strings.HasPrefix(v.GetE2EImage(), i) {
+				continue imageLoop
+			}
+		}
+
```

Comment:
I'm not 100% sure we should do this here. In my mind, the images command isn't to be used exclusively for image mirroring. The invalid image is in fact used by tests, it just doesn't exist.

I feel like we should stick to the output of `GetE2EImage()` and filter this out somewhere else. I don't have strong feeling, though.